packages/widgets/src/lib/components/InferenceWidget/shared/WidgetWrapper/WidgetWrapper.svelte

```diff
@@ -35,6 +35,10 @@
 			if (modelLoadInfo?.state === "TooBig") {
 				updateWidgetState(model.id, "isDisabled", true);
 			}
+
+			if (!$tgiSupportedModels) {
+				tgiSupportedModels.set(await getTgiSupportedModels(apiUrl));
```

Comment:
```suggestion
				$tgiSupportedModels await getTgiSupportedModels(apiUrl);
```
I think we can use this syntax for setting as well